AdminWebsite/AdminWebsite.AcceptanceTests/Steps/BookingDetailsSteps.cs

```diff
@@ -98,10 +99,46 @@ private void VerifyJudgeInParticipantsList()
             _browsers[_c.CurrentUser].Driver.WaitUntilVisible(BookingDetailsPage.JudgeName).Text.Should().Contain(judge.DisplayName);
 
             if (!OnlyDisplayEmailAndUsernameIfCurrentUserMadeTheBooking()) return;
-            _browsers[_c.CurrentUser].Driver.WaitUntilVisible(BookingDetailsPage.ParticipantEmail(hearingJudge.Id)).Text.Should().Be(judge.AlternativeEmail);
+            if (GetJudgeEmail(hearing) != null)
+            {
+                _browsers[_c.CurrentUser].Driver.WaitUntilVisible(BookingDetailsPage.ParticipantEmail(hearingJudge.Id)).Text.Should().Be(judge.AlternativeEmail);
+            }
+            else
+            {
+                _browsers[_c.CurrentUser].Driver.WaitUntilVisible(BookingDetailsPage.ParticipantEmail(hearingJudge.Id)).Text.Should().Be("TBC");
+            }
             _browsers[_c.CurrentUser].Driver.WaitUntilVisible(BookingDetailsPage.ParticipantUsername(hearingJudge.Id)).Text.Should().Be(judge.Username);
         }
 
+        private string GetJudgeEmail(HearingDetailsResponse hearing)
+        {
+            var email = GetOtherInformationObject(hearing.OtherInformation).JudgeEmail;
+            if (email == string.Empty)
+            {
+                return null;
+            }
+            return email;
+        }
+        
+        private static OtherInformationDetails GetOtherInformationObject(string otherInformation)
+        {
+            try
+            {
+                var properties = otherInformation.Split("|");
+                return new OtherInformationDetails
+                {
+                    JudgeEmail = properties[2],
+                    JudgePhone = properties[4],
+                    OtherInformation = properties[6]
+                };
+            }
+            catch (Exception)
+            {
+                var properties = otherInformation.Split("|");
+                return new OtherInformationDetails {OtherInformation = properties[2]};
```

Comment:
where did 2 come from? likely to come here because its not using this new format and doesnt contain any `|` ? properties[2] would just throw another our of range exception?

Reply:
The old format would be parsed to `|OtherInformation|Sometext|`, so:
[0] = empty string
[1] = OtherInformation
[2] = Sometext --> We want this value

Reply:
oh, is there a migration to change the old information to this format?

Reply:
I'm unsure, I think that may have been part of David's work? This is the format I was getting back whilst testing

Reply:
can we add a length check like the other to be safe?